packages/mui-lab/src/LoadingButton/LoadingButton.test.js

```diff
@@ -72,4 +73,44 @@ describe('<LoadingButton />', () => {
       expect(screen.getByRole('button')).to.have.text('loading…Test');
     });
   });
+
+  describe('ButtonGroup works with LoadingButton', () => {
+    it('correctly passes props to children', () => {
+      const { getByRole } = render(
+        <ButtonGroup variant="contained" size="large" color="secondary">
```

Comment:
```suggestion
        <ButtonGroup variant="outlined" size="large" color="secondary">
```

Let's use non-default variant value.